### PR TITLE
feat: Add graph description as tooltip content over y-label

### PR DESCRIPTION
**Pull request by schakrad**
Screenshot 2023-10-11 at 10 10 27 AM

### CODE REVIEW

**Comment by ashutosh16** [comment]
Pls, add a video or screenshot of the expected.

**Comment by schakrad** [comment]
> Pls, add a video or screenshot of the expected.

Added the screenshot.

File: extensions/resource-metrics/resource-metrics-extention/ui/src/Metrics/Chart/AnomalyChart.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { useMemo } from "react";
+import {useMemo, useState} from "react";
```
Comment (ashutosh16) [inline extensions/resource-metrics/resource-metrics-extention/ui/src/Metrics/Chart/AnomalyChart.tsx:1]: Why are you using `useState`? Do you want to show tooltip once?

File: extensions/resource-metrics/resource-metrics-extention/ui/src/Metrics/Chart/AnomalyChart.tsx [hunk 2]
```diff
@@ -601,7 +631,7 @@ export const AnomalyChart = ({
               />
               {YAxisMemo}
               {TooltipMemo}
-              {LegendMemo}
+              {thisChartData?.[0]?.data.length >0 ? !isLabelHovered ? LegendMemo : null : null}
```
Comment (ashutosh16) [inline extensions/resource-metrics/resource-metrics-extention/ui/src/Metrics/Chart/AnomalyChart.tsx]: Would this work?
```suggestion
{thisChartData?.[0]?.data.length > 0 && !isLabelHovered && LegendMemo}
``````

File: extensions/resource-metrics/resource-metrics-extention/ui/src/Metrics/Chart/AnomalyChart.tsx (hunk 1 above)
Comment (schakrad) [reply extensions/resource-metrics/resource-metrics-extention/ui/src/Metrics/Chart/AnomalyChart.tsx:1]: I am using useState for displaying the tooltip over the ylabel. Without state, when I implement tooltip, the legendmemo which acts as tooltip and the ylabel tooltip are  both popping (2 tooltips getting displayed). I need to disable the legend-memo tooltip when user hovers over ylabel. The legendMemo tooltip gets displaayed when usr hovers over any part of graph. I need to stop this behaviour when user hovers over ylabel.

**Review by ashutosh16 — APPROVED** [review]
LGTM

**Pull request merged**